packages/common/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@dsco_/common",
+  "version": "1.0.0",
+  "description": "Common styles and utilities for DSCO consumers",
+  "author": "Madelyn Kasula <maddie.kasula@code.org>",
+  "homepage": "https://github.com/code-dot-org/dsco_/tree/main/packages/common#readme",
+  "license": "MIT",
+  "exports": {
+    "./styles": "./styles/_styles.scss",
+    "./tokens": "./styles/_tokens.scss"
+  },
```

Comment:
note: this package uses "exports" to define entrypoints and doesn't have a "build" script, so it is not built by webpack -- consumers are responsible for that.

i expect we'll eventually add JavaScript to this package (like a `<ThemeProvider>`?) and will build certain directories, but will leave these exports.